windows/README.md

```diff
@@ -0,0 +1,40 @@
+# Windows Powershell Scripts
+
+
+### Get SmartFace services
+```
+Get-Service -Name "SF*"
+```
+
+****************************************************************************************************************************************************************************
+**WARNING!**
+  
+Scripts query services by the Service Name pattern. 
+It may happen that other application/system register service with similair pattern. 
+We always recomended you to get all SmartFace services list before to ensure only propper SmartFace services will be affected.
```

Comment:
recommendation: `We always recommend you to list all SmartFace services before running other commands to ensure that only SmartFace are affected.`